static/js/narrow.js

```diff
@@ -744,7 +746,9 @@ function handle_post_narrow_deactivate_processes() {
     message_edit.handle_narrow_deactivated();
     widgetize.set_widgets_for_list();
     typing_events.render_notifications_for_narrow();
-    tab_bar.initialize();
+    if (!page_params.search_pills_enabled) {
+        tab_bar.initialize();
+    }
```

Comment:
Can you explain why this change is correct?  I would have expected we'd still need to initialize the tab_bar module?

Reply:
I made this change as there were 2 searchboxes displayed when pills were enabled, I think the first commit could be considered as a non-mergable testing commit.

Reply:
OK, makes sense.  Best to label it as such clearly, e.g. "TESTING" in the commit message title.